tutorial/_toc.yml

```diff
@@ -0,0 +1,19 @@
+# Table of contents
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: home
+parts:
+  - caption: Getting started
+    chapters:
+      - file: getting_started/download_tutorial
+      - file: getting_started/install_dependencies
+      - file: getting_started/launch_notebooks
+  - caption: Tutorial notebooks
+    chapters:
+    - file: notebooks/images_are_arrays
+    - file: notebooks/image_filtering
+    - file: notebooks/segmentation_and_measuring
```

Comment:
I created these new empty notebooks to reflect the structure proposed in https://cfp.scipy.org/2023/talk/NEUUKG/. I've left the old tutorial notebooks for now in case we wan to refer to them.